### ISSUE TITLE
Make library more robust by using a list of known nanopub servers with grlc endpoints

### ISSUE THREAD

**Issue by raar1** [fair-workflows/nanopub#13]
From [here](https://github.com/peta-pico/nanobench/blob/1ea520298266d0961d420f22fc7538f951bbaed7/src/main/java/org/petapico/nanobench/ApiCall.java#L41):
```
	public static String[] apiInstances = new String[] {
		"http://grlc.nanopubs.lod.labs.vu.nl/api/local/local/",
		"http://130.60.24.146:7881/api/local/local/",
		"https://openphacts.cs.man.ac.uk/nanopub/grlc/api/local/local/",
		"https://grlc.nanopubs.knows.idlab.ugent.be/api/local/local/",
		"http://grlc.np.scify.org/api/local/local/",
		"http://grlc.np.dumontierlab.com/api/local/local/"
	};
```
The above are all nanopub servers with grlc endpoints. Currently we only use the VU one, but we should have a list of these servers to try in the event that some are down. In future it is expected that the addition of new nanopub servers will be published as nanopubs themselves, so we can consider also having a ```look_for_new_nanopub_servers``` kind of method that can update the local list. We don't yet know what these nanopubs will look like though, so that will have to wait.

**Comment by svenvanderburg** [comment]
https://github.com/fair-workflows/nanopub/pull/21

**Comment by svenvanderburg** [comment]
Fixed by #21
